### PR TITLE
Updated packages related to eslint

### PR DESCRIPTION
**Pull request by AleshaOleg**

### CODE REVIEW

**Comment by AleshaOleg** [comment]
@kof but there's no prettier in this project

**Comment by kof** [comment]
https://github.com/cssinjs/jss/blob/master/.prettierrc

**Comment by kof** [comment]
feel free to add it

**Comment by AleshaOleg** [comment]
ok

**Comment by AleshaOleg** [comment]
@kof done

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+node_modules/
+dist/
+lib/
+*.html
+*.md
```
Comment (kof) [inline .prettierignore]: why? can be prettified I think we do it in the jss repo

Comment (AleshaOleg) [reply .prettierignore]: Yes, you're doing. There's just one .md file, so I think there's no reason to include this

**Pull request merged**